sonar-csharp-plugin/pom.xml

```diff
@@ -155,7 +155,8 @@
           <pluginClass>org.sonar.plugins.csharp.CSharpPlugin</pluginClass>
           <skipDependenciesPackaging>true</skipDependenciesPackaging>
           <sonarLintSupported>false</sonarLintSupported>
-          <sonarQubeMinVersion>${sonarLTS.version}</sonarQubeMinVersion> <!-- runtime minimal API version -->
+          <pluginApiMinVersion>${sonarLTS.version}</pluginApiMinVersion> <!-- runtime minimal API version -->
```

Comment:
pluginApiMinVersion is not the same thing as SQ version. They have independent lifecycles now. 

Hardcode the value here

Reply:
I think API version is in `sonar.version` property. Can you check with Cody if that's the case and having `10.1.0.809` would work here?

sonar-dotnet doesn't care about MIN Api version, because we do not run in Java SonarLint flavors, and we always run under the latest release of SQ/SC.

Reply:
I think it should be fine. In any case, it should not affect the on-demand downloading feature.